resources/lib/globals.py

```diff
@@ -215,6 +215,17 @@ def init_globals(self, argv):
         self.TIME_TRACE_ENABLED = self.ADDON.getSettingBool('enable_timing')
         self.IPC_OVER_HTTP = self.ADDON.getSettingBool('enable_ipc_over_http')
 
+        import resources.lib.database.db_local as db_local
+        self.LOCAL_DB = db_local.NFLocalDatabase()
+        import resources.lib.database.db_shared as db_shared
+        import resources.lib.database.db_utils as db_utils
+        # TODO: xml settings to specify a custom path
+        #  need to study how better apply to client/service
```

Comment:
For the client, it's not a real problem IMO as it will be checked at every lanch.

For the service, what I did for the library updater is to make it a xbmc.Monitor instance, and compute the time again at event 'onSettingsChanged'. Maybe it can be done the same way for all services?

Or we need to send an AddonSignal when settings_monitor detects a setting change.